Chat/File.cpp

```diff
@@ -29,20 +35,40 @@ bool File::SetFile(const std::string &path)
         std::getline(str_stream, name_, '\\');
     }
 
+    HANDLE file = CreateFile(path_.c_str(),             //path
```

Comment:
I dont mean that path may be invalid. I say that function may fails by some reasons (it may work good and never fails but in one day it would fail by some reasons). Just please put checking for good HANDLE after function.